src/plugin.js

```diff
@@ -146,6 +146,12 @@ export default function installYamcsPlugin(configuration) {
             cssClass: "icon-generator-events"
         });
 
+        openmct.types.addType(OBJECT_TYPES.COMMANDS_OBJECT_TYPE, {
+            name: "Commands",
+            description: "To view command history",
+            cssClass: "icon-generator-events" // TODO: replace
```

Comment:
Please file a followup issue and assign to Charles and Rukmini.

Reply:
@akhenry 

Follow-up issue is here: https://github.com/akhenry/openmct-yamcs/issues/186

But I think we may need to add Rukmini and Charles as collaborators to this repo? I'm unable to assign to them.